src/views/Dashboard/components/DashboardCards/SavedFilters/ListItem/index.tsx

```diff
@@ -39,22 +40,17 @@ const SavedFiltersListItem = ({ data }: OwnProps) => {
             onOk: () => dispatch(deleteSavedFilter(data.id)),
           })
         }
-        linkProps={{
-          to: {
-            pathname: FILTER_TAG_PAGE_MAPPING[data.tag],
-            search: `?${FILTER_ID_QUERY_PARAM_KEY}=${data.id}`,
-          },
-          content: (
-            <Text style={{ width: 400 }} ellipsis={{ tooltip: data.title }}>
-              {data.title}
-            </Text>
-          ),
-          onClick: () =>
-            setQueryBuilderState(FILTER_TAG_QB_ID_MAPPING[data.tag], {
-              active: data.queries[0].id,
-              state: data.queries,
-            }),
+        onClick={() => {
+          history.push(
+            `${FILTER_TAG_PAGE_MAPPING[data.tag]}?${FILTER_ID_QUERY_PARAM_KEY}=${data.id}`,
+          );
+
+          setQueryBuilderState(FILTER_TAG_QB_ID_MAPPING[data.tag], {
+            active: data.queries[0].id,
```

Comment:
do we expect `data.queries[0]` to always exist? If yes, leave it as it is if not, add an `?`